File: exs/mundo_1/c#/009.cs
// faça um programa que leia um número inteiro qualquer e mostre na tela a sua tabuada

using System;

static class io {
    public static string input(string text) {
        Console.Write(text);
        return Console.ReadLine();
    }
    
    public static void print(object data, string end="\n") {
        string text = data.ToString();

        Console.Write($"{text}{end}");
    }
}

class Program {
    public static void Main() {
      int valor = int.Parse(io.input("Digite um número inteiro qualquer: "));

      io.print($"{valor} x 01 = {valor*1}");
      io.print($"{valor} x 02 = {valor*2}");
      io.print($"{valor} x 03 = {valor*3}");
      io.print($"{valor} x 04 = {valor*4}");
      io.print($"{valor} x 05 = {valor*5}");
      io.print($"{valor} x 06 = {valor*6}");
      io.print($"{valor} x 07 = {valor*7}");
      io.print($"{valor} x 08 = {valor*8}");
      io.print($"{valor} x 09 = {valor*9}");
      io.print($"{valor} x 10 = {valor*10}");
    }
}

File: exs/mundo_1/rust/009.rs
/*
Desafio 009

Problema: Faça um programa que leia um número Inteiro qualquer e mostre na tela a sua
          tabuada.
*/

fn main() {
    let mut valor = String::new();

    println!("Digite um número: ");
    std::io::stdin().read_line(&mut valor).unwrap();

    let valor: i32 = valor.trim().parse().expect("Erro ao ler valor");

    println!("{} x  1 = {}", valor, valor * 1);
    println!("{} x  2 = {}", valor, valor * 2);
    println!("{} x  3 = {}", valor, valor * 3);
    println!("{} x  4 = {}", valor, valor * 4);
    println!("{} x  5 = {}", valor, valor * 5);
    println!("{} x  6 = {}", valor, valor * 6);
    println!("{} x  7 = {}", valor, valor * 7);
    println!("{} x  8 = {}", valor, valor * 8);
    println!("{} x  9 = {}", valor, valor * 9);
    println!("{} x 10 = {}", valor, valor * 10);
}
